lib/taskana-core/src/main/java/pro/taskana/jobs/JobRunner.java

```diff
@@ -100,7 +100,8 @@ private void runJobTransactionally(ScheduledJob scheduledJob) {
         } catch (Exception e) {
             e.printStackTrace();
```

Comment:
The exception should be logged by the logger

Reply:
In this case, the error was logged as warn error level